grails-app/conf/Config.groovy

```diff
@@ -1081,6 +1081,9 @@ grails.plugin.raven.dsn = "https://{PUBLIC_KEY}:{SECRET_KEY}@app.getsentry.com/{
 // Default Ajax request timeout
 openboxes.ajaxRequest.timeout = 120000
 
+// connection timeout milliseconds
+openboxes.connection.timeout = 8000
```

Comment:
Can you explain the rationale behind the 8 second timeout? Rather than something round like 5 or 10 seconds?

Also let's be a little more explicit about what this property is for. "connection" is an overloaded term - in the context of our application it could mean database, URL, and someday maybe websocket, etc

So let's go with something more descriptive.
```
// Timeout used with the navigator.onLine event listener
openboxes.browser.connection.status.timeout = 8000
```
Or if that's still too vague, we really only care about this from the perspective that it tells us whether the application is online or offline (whichever makes more sense)

* openboxes.online.status.listener.timeout
* openboxes.offline.status.listener.timeout
* openboxes.browser.online.listener.timeout

Or we could map it to the specific API we're using. I don't love this since the API could change in the future, but it's slightly better than "openboxes.connection.timeout"

* openboxes.navigator.online.timeout

Reply:
You are right, looking at it again I should have picked a better name for it, I was inspired by the variable name above for `ajaxRequest.timeout`.

`openboxes.browser.connection.status.timeout` sounds good

As for the reasoning behind _8000 ms_ for timeout I don't have a strong opinion about it and we can discuss what is the most optimal value.